### ISSUE TITLE
Automated deployment

### ISSUE THREAD

**Issue by raphaelmue** [raphaelmue/dashup#74]
When the master branch is merged into the deployment branch, after pushing the branch, the Jenkins pipeline should start the application on the server. This process has to be automated.

**Comment by raphaelmue** [comment]
We need to implement that the selenium tests run on a random port, so that the application can run in parallel. If tests fail, then the application will be stopped and won't be started again.

**Comment by raphaelmue** [comment]
> Ill start that now... ill try to implement that fast cause all tests are failing currently

Opened a new issue (#77) and renamed the branch. If this issue is fixed, this issue can be continued.
